### DISCUSSION TITLE
Windows 11 stuck on startup.nsh (Mac M1 Monterey)

### DISCUSSION THREAD

**Discussion by nolanseymour22** [utmapp/UTM#3798] [UTM Help (macOS)]
I've download UTM and Window11 Insider Preview.
I have tried on multiple versions of UTM to boot up the Windows 11 ARM 64 VM, however I am always stuck on startup.nsh.
I have followed several tutorials on how to install.
Whenever I create the new VM after booting the Windows11 Insider ARM 64 VHDX file,  the architecture listed for it in is x86_64, and when I try to change it to ARM64, the VM just does not start up at all. Please if anyone can help that would be great. Attached is a short recording of my issue.

https://drive.google.com/file/d/1-j27c5lAHZ2yokdWDu_n9aWrExDTHu_v/view?usp=sharing
![58f4b24f8fb41_QQ20170417201516 png 0b5197765393276a78418504779a333f](https://user-images.githubusercontent.com/93801271/159219397-359cc2a7-4d02-4969-9097-a9dfea71cd8c.png)

**Comment by Qhilm** [comment]
Same issue with:
* ISO fetched using CrystalFetch (Apple Silicon, Windows 11, US EN)
* UTM 4.3.5 (87)
* MacBook Pro M1 (2021)

<img width="1064" alt="image" src="https://github.com/utmapp/UTM/assets/3350433/e0613ec0-4790-434c-8bb2-94508e1826d5">

Did you resolve your issue @nolanseymour22 ? If yes, do you mind sharing how?
